src/libraries/icubmod/embObjMotionControl/embObjMotionControl.cpp

```diff
@@ -3269,7 +3321,15 @@ bool embObjMotionControl::getKinematicMJRaw(int j, double &rotres)
     return false;
 }
 
-bool embObjMotionControl::getHasTempSensorsRaw(int j, int& ret)
+bool embObjMotionControl::getTemperatureSensorTypeRaw(int j, int& ret)
```

Comment:
Since you added this feature, I suggest returning the type as a string and not as int so it is more clear to the user.